src/e2e-test/features/servicenow/RunTimeWithMacros.feature

```diff
@@ -84,3 +84,100 @@ Feature: ServiceNow Source - Run time scenarios (macro)
     And Wait till pipeline is in running state
     And Open and capture logs
     Then Verify the pipeline status is "Succeeded"
+
```

Comment:
let's rename this as we now have sink and multisource tests : servicenow -> servicenowsource

Reply:
Done